hail_search/requirements-test.in

```diff
@@ -1,2 +1,3 @@
 coverage<5.2
 pytest-aiohttp
+google-cloud-storage>=2.14.0
```

Comment:
what in the hail search tests relies on cloud storage? The tests were passing before this commit, and I don'tt think. its neccessary to add this dependency

Reply:
Yeah, I don't think we need this dependency on the seqr side.  This dependency is our gencode download in the pipeline.

Reply:
oh I was literally looking at a unit test run from another repo, my bad I'll remove this